### PR TITLE
Support default fields

### PR DESCRIPTION
**Pull request by davisshaver**
See: https://github.com/fusioneng/Shortcode-UI/issues/12

### CODE REVIEW

**Comment by davisshaver** [comment]
@danielbachhuber, could I get a check on first field addition?

Here’s how it’s invoked in shortcode manager:
`'superelement' => array(
            'label' => 'Superelement',
            'listItemImage' => 'dashicons-smiley',
            'attrs' => array(
                array(
                    'label' => 'Select',
                    'attr'  => 'select',
                    'type'  => 'select',
                    'options' => array('value1' => 'label1', 'value2' => 'label2')
                ),
                array(
                    'label' => 'Radio',
                    'attr'  => 'radio',
                    'type'  => 'radio',
                    'options' => array('value1' => 'label1', 'value2' => 'label2')
                ),
            ),`

UI
screen shot 2014-11-03 at 3 14 27 pm

Editor
screen shot 2014-11-03 at 3 14 50 pm

File: js/shortcode-ui.js [hunk 1]
```diff
@@ -206,6 +206,11 @@ var Shortcode_UI;
 			'change select':               'updateValue',
 			'change input[type=checkbox]': 'updateValue',
 			'change input[type=radio]':    'updateValue',
+			'change input[type=email]':    'updateValue',
+			'change input[type=number]':    'updateValue',
+			'change input[type=hidden]':    'updateValue',
```
Comment (davisshaver) [inline js/shortcode-ui.js]: @danielbachhuber, this doesn’t work as expected because the hidden input doesn’t ever change. Suggestions?

Comment (danielbachhuber) [inline js/shortcode-ui.js]: :facepunch: :football:

**Comment by davisshaver** [comment]
screen shot 2014-11-03 at 4 53 14 pm

screen shot 2014-11-03 at 4 53 03 pm

**Comment by davisshaver** [comment]
@danielbachhuber, removed hidden input type support for now. Could you take a pass through this when you get a chance?

**Pull request merged**